app/_locales/pt_BR/messages.json

```
@@ -128,44 +203,53 @@
     "message": "O saldo pode estar desatualizado"
   },
   "basic": {
-    "message": "Básicas"
+    "message": "Básico"
+  },
+  "blockExplorerUrl": {
+    "message": "Block Explorer"
   },
   "blockExplorerView": {
-    "message": "Ver conta em $1",
+    "message": "Exibir conta em $1",
     "description": "$1 replaced by URL for custom block explorer"
   },
   "blockiesIdenticon": {
-    "message": "Usar o Identicon de Blocos"
+    "message": "Usar identicon de Blockies"
   },
   "browserNotSupported": {
     "message": "Seu navegador não é compatível..."
   },
   "builtInCalifornia": {
-    "message": "O MetaMask foi projetado e desenvolvido na Califórnia."
+    "message": "O MetaMask é projetado e construído na Califórnia."
+  },
+  "buy": {
+    "message": "Comprar"
   },
   "buyWithWyre": {
     "message": "Comprar ETH com Wyre"
   },
   "buyWithWyreDescription": {
-    "message": "O Wyre permite que você utilize um cartão de crédito para depositar ETH direto na sua conta do MetaMask."
+    "message": "Com o Wyre, você pode usar um cartão de débito para depositar ETH diretamente na sua conta do MetaMask."
+  },
+  "bytes": {
+    "message": "Bytes"
   },
   "cancel": {
     "message": "Cancelar"
   },
   "cancellationGasFee": {
-    "message": "Tarifa de Gas de cancelamento"
+    "message": "Taxa em gas de cancelamento"
```

This is a bit weird to read. I'd say that either we specify the unit between parentheses, or we leave it to the end of the sentence.

```
Taxa (em gas) de cancelamento
```

Or 

```
Taxa de cancelamento em gas
```

---

Thanks! I'll take the second suggestion for now.

```suggestion
    "message": "Taxa de cancelamento em gas"
```